src/confopt/train/experiment.py

```diff
@@ -119,6 +154,30 @@ def train_supernet(
         exp_runtime_to_load: str | None = None,
         use_benchmark: bool = False,
     ) -> ConfigurableTrainer:
+        """Trains a supernet using the given profile with options for loading previous
+        runs.
+
+        Args:
+            profile (BaseProfile): Contains configurations for training the supernet,
+            including compnent settings and architectural specifications.
+
+            model_to_load (str | int | None): Specifies the training state to load the
+            supernet from. Valid values are "last" or "best", representing the most
+            recent or the best-performing model checkpoint, respectively.
+            If an integer is provided, it represents the epoch number from which
+            training should be continued.
+            If `None`, then it would start a new run.
+
+            exp_runtime_to_load (str | None): The particular experiment runtime to
+            load the model from.If `None`, the model will be loaded from the last
+            runtime.
+
+            use_benchmark (bool): If `True`, uses a benchmark API for evaluation,
```

Comment:
"instead of training the model from scratch" is misleading because it implies that if `use_benchmark` were False, it would train models from scratch to evaluate intermediate results.